src/main/java/com/shigotosagashi/jobdomainservice/dto/in/JobIn.java

```diff
@@ -0,0 +1,49 @@
+package com.shigotosagashi.jobdomainservice.dto.in;
+
+import com.shigotosagashi.jobdomainservice.domain.JobEntity;
+
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.Random;
+import java.util.UUID;
+
+public class JobIn {
+  @NotNull(message = "Missing required field title")
+  public final String title;
+  @NotNull(message = "Missing required field description")
+  public final String description;
+  public final Optional<Long> creationDate;
+  @NotNull(message = "Missing required field expirationDate")
+  public final long expirationDate;
```

Comment:
For DTOs, prefer using wrappers. Use Long

Reply:
Corrected